src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -21,10 +21,26 @@
     @Column(name = "CART_ID")
     private long id;
 
-    @OneToOne
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
Tu zdecydowanie nie powinno być cascadeType all. Oznaczało by to, że usunięcie koszyka powinno propagować np. usunięcie usera. Lepiej w testach zapisać po prostu osobno usera, a nie używać tutaj w ogóle cascady.